test/project_types/script/commands/push_test.rb

```diff
@@ -53,19 +60,19 @@ def test_push_propagates_error_when_ensure_env_fails
 
       def test_does_not_force_push_if_user_env_already_existed
         @force = false
-        Layers::Application::PushScript.expects(:call).with(ctx: @context, force: @force)
+        Layers::Application::ConnectApp.expects(:call).returns(false)
+        Layers::Application::PushScript.expects(:call).with(ctx: @context, force: false)
```

Comment:
Nitpick: do you still need line 62?

Reply:
Yes. The `@force` here is the simulated command-line-argument used when performing the command. I don't want _that_ `@force` to change the outcome of the tests.